logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/ConversationEventReceiver.kt

```diff
@@ -271,12 +272,23 @@ class ConversationEventReceiverImpl(
                 )
             }
 
+    private suspend fun handleDeletedConversation(event: Event.Conversation.DeletedConversation) =
+        messageRepository.deleteAllMessagesForConversation(event.conversationId)
```

Comment:
What's the reason for deleting only the messages and not the conversation it self? Do we still want to show deleted conversations in the conversation list?

Reply:
I was doing it in a 2-step process, but ended up going for the `ON DELETE CASCADE`, simplifying the code.